recipe/meta.yaml

```diff
@@ -377,7 +377,6 @@ outputs:
         - libxml2-devel
         - zlib
         - zstd
-        - compiler-rt{{ major_version }} {{ version }}
```

Comment:
How does removing a host-dep reduce clobbering (given that host-deps form the baseline for the content of `$PREFIX` that's compared before/after installation)? If anything, the build here will pick up more files now rather than less...?